ports/quirc/CMakeLists.txt

```
@@ -1,17 +1,27 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.19)

```

```suggestion
cmake_minimum_required(VERSION 3.7)
```
is probably more than enough considering the "complexity" of the CMakeLists.txt :)
vcpkg internally uses 3.19 but users might not be using that version (especially on unix)

---

Right, but this is being built by vcpkg so presumably the version users use doesn't matter?